src/modules/shared/UploadBucket.tsx

```diff
@@ -57,6 +57,7 @@ export const UploadBucket = ({
           component='span'
           isLoading={isLoading}
           startIcon={<FileUploadIcon />}
+          sx={{ marginTop: 4 }} 
```

Comment:
Inline styles should be avoided

Reply:
We need some spacing between the form and the upload functionality. I thing spacing classes should be applied to the container `<Box />`